### ISSUE TITLE
Error: spawn UNKNOWN error when linking (Windows)

### ISSUE THREAD

**Issue by aaronsturm** [transistorsoft/react-native-background-geolocation#312]
I was following the instructions here: https://github.com/transistorsoft/react-native-background-geolocation/blob/master/docs/INSTALL-ANDROID-RNPM.md

```
$ react-native link react-native-background-geolocation
Scanning 586 folders for symlinks in E:\Sites\Skytango\app\node_modules (30ms)
rnpm-install info Linking react-native-background-geolocation android dependency
rnpm-install info Android module react-native-background-geolocation has been successfully linked
rnpm-install info Linking react-native-background-geolocation ios dependency
rnpm-install WARN ERRGROUP Group 'Frameworks' does not exist in your Xcode project. We have created it automatically for you.
rnpm-install info iOS module react-native-background-geolocation has been successfully linked
rnpm-install ERR! Something went wrong while linking. Error: spawn UNKNOWN
Please file an issue here: https://github.com/facebook/react-native/issues

```
## Your Environment
* Plugin version:
2.9.2
* Platform:
Android
* OS version:
Windows 10
* Device manufacturer / model:
n/a just using the emulator
* React Native version (`react-native -v`):
react-native-cli: 2.0.1
react-native: 0.47.2

* Plugin config

## Expected Behavior
My expectations are that it would run the link command without producing an error.

## Actual Behavior
Received this error in the command output:
```
rnpm-install ERR! Something went wrong while linking. Error: spawn UNKNOWN
Please file an issue here: https://github.com/facebook/react-native/issues

```
## Steps to Reproduce

1. npm install --save react-native-background-geolocation
2. react-native link react-native-background-geolocation

## Context
I was trying to do the initial first time install of this package to see if it would fit my needs.

## Debug logs
No additional log output was created when running the command.

Thanks!

**Comment by christocracy** [comment]
These react-native link scripts are quite new and haven't been tested on Windows.

They work fine on Mac OS
```
ford-2:Foo5 chris$ react-native link react-native-background-geolocation
Scanning 582 folders for symlinks in /Users/chris/Documents/workspace/react/background-geolocation/tmp/Foo5/node_modules (3ms)
rnpm-install info Linking react-native-background-geolocation android dependency
rnpm-install info Android module react-native-background-geolocation has been successfully linked
rnpm-install info Linking react-native-background-geolocation ios dependency
rnpm-install WARN ERRGROUP Group 'Frameworks' does not exist in your Xcode project. We have created it automatically for you.
rnpm-install info iOS module react-native-background-geolocation has been successfully linked
```

You'll have to do the Manual Setup until I can test on a Windows VM.

**Comment by aaronsturm** [comment]
Ok, I'm doing that now. Thanks.

**Comment by aaronsturm** [comment]
I forgot to follow-up. The manual setup did work.

**Comment by christocracy** [comment]
Thanks for following up

**Comment by christocracy** [comment]
However, I need to test the script on Windows to see what's up, so I'll keep this open

**Comment by ntomallen** [comment]
I just got this issue as well (but didn't get the error when running link with other packages.) I'm on the Fall Creator's update of Windows 10.

Here is the output after running it a second time. It appears that the iOS module gets linked but not the Android?
```
rnpm-install info Linking react-native-background-geolocation android dependency
rnpm-install info Android module react-native-background-geolocation has been successfully linked
rnpm-install info iOS module react-native-background-geolocation is already linked
rnpm-install ERR! Something went wrong while linking. Error: spawn UNKNOWN
Please file an issue here: https://github.com/facebook/react-native/issues
```
Anyways, I'll try the manual setup.

**Comment by djereg** [comment]
Fix submitted in #598
